packages/graphiql/src/components/GraphiQL.js

```diff
@@ -39,6 +39,19 @@ import {
 
 const DEFAULT_DOC_EXPLORER_WIDTH = 350;
 
+// eslint-disable-next-line no-console
+const logger = console.log;
+
+if (!React.version || !React.version.indexOf('16') < 0) {
```

Comment:
Is `React.version` a public API? I couldn't find it in the React docs (didn't look for long though)

Reply:
yes it’s widely used

Reply:
![9674C3E4-58C7-4A4A-A687-FAF0EAF2C683](https://user-images.githubusercontent.com/1368727/71025916-1673cf00-20d6-11ea-980a-81f284f063fa.jpeg)

from very early on its been an available APi!